### ISSUE TITLE
References and design doc request

### ISSUE THREAD

**Issue by Padarn** [borglab/gtsfm#212]
Hi all,

I learnt about this library from a talk I wanted from the TUM AI lecture series. It looks very promising.

However, being fairly new to the field, it would be great if you could point to some references to follow to understand the algorithms and designs of GTSFM and how it compares to other available packages.

Thanks!

**Comment by johnwlambert** [comment]
Hi @Padarn, thanks for your interest in our work. I would start with the references linked for each module in the Readme: https://github.com/borglab/gtsfm#repository-structure.

A more formal design doc/whitepaper is still forthcoming.

**Comment by Padarn** [comment]
Thanks @johnwlambert I will do. 

Maybe I can ask a more specific question then: It is really cool to be able to natively support Dask and distributed computation, but I'm trying to find some motivation for running large scale SFM rather than breaking a dataset down and doing it in batches (organized into areas for example). I know its a vague question, but maybe you have some ideas on how to measure the quality of a reconstruction vs the number of images involved.

**Comment by johnwlambert** [comment]
parallelism through Dask and hierarchical SfM are by no means contradictory to each other :-)

**Comment by Padarn** [comment]
Right, I meant something even less intelligent than hierarchical :-) For example, just building a fully partitioned problem (which you could also use dask for, but it would also be embarrassingly parallel so you wouldn't really need to).
